docs/data/base/components/menu/UseMenu.js

```diff
@@ -44,6 +44,7 @@ const styles = `
     padding: 8px;
     border-radius: 0.45em;
     cursor: default;
+    user-select: none;
   }
 
```

Comment:
Unrelated to these changes but we should probably make this one, in terms of item hovering, work in dark mode too :)

<img width="225" alt="Screen Shot 2022-05-11 at 02 27 09" src="https://user-images.githubusercontent.com/67129314/167775306-6051d1c4-2a93-4b53-a027-19ac9c821a2f.png">

Reply:
Good catch! I'll fix it.
I suppose I'll have to turn on the dark mode on our docs, so I spot these issues earlier 😅